server/src/main/java/com/paypal/selion/grid/ProcessLauncherOptions.java

```diff
@@ -0,0 +1,160 @@
+package com.paypal.selion.grid;
+
+import com.paypal.selion.SeLionConstants;
+import com.paypal.selion.pojos.SeLionGridConstants;
+
+/**
+ * These options give a programmatic way to influence some of the behaviors of a process launcher. These behaviors may
+ * or may not have a SeLion Grid dash argument that corresponds to them. Dash arguments override ProcessLauncherOptions.
+ */
+public final class ProcessLauncherOptions {
+    private boolean includeSystemProperties = true;
+    private boolean includeParentProcessClassPath = true;
+    private boolean includeJarsInSeLionHomeDir = true;
+    private boolean includeJarsInPresentWorkingDir = true;
+    private boolean setupLoggingForJavaSubProcess = true;
+    private boolean includeWebDriverBinaryPaths = true;
+    private boolean continuousRestart = true;
+    private boolean fileDownloadCleanupOnInvocation = true;
+    private boolean fileDownloadCheckTimeStampOnInvocation = true;
+
+    /**
+     * Enable/Disable clean up of previously downloaded artifacts for subsequent calls to {@link FileDownloader} .
+     * Default: enabled.
+     */
+    public ProcessLauncherOptions setFileDownloadCleanupOnInvocation(boolean val) {
```

Comment:
Since the method intends to `enabled/disable`, should we rename this method to `toggleFileDownloadCleanupOnInvocation` ?